examples/sw/simple_system/common/link.ld

```diff
@@ -6,9 +6,9 @@ OUTPUT_ARCH(riscv)
 
 MEMORY
 {
-    /* 992 kB should be enough for anybody... */
-    ram         : ORIGIN = 0x00100000, LENGTH = 0xF8000 /* 992 kB */
-    stack       : ORIGIN = 0x001F8000, LENGTH = 0x8000  /* 32 kB */
+    /* Change this if you'd like different sizes */
```

Comment:
I think it would be more useful if the comment would say something like: "The Arty A7 FPGA board offers 256 kB of BRAM."

Reply:
I agree, thanks for the comment. I tried to update it with more detail.